installer_scripts/install-yb-voyager

```diff
@@ -258,9 +258,11 @@ install_ora2pg() {
 	sudo cpanm DBD::mysql Test::NoWarnings DBD::Oracle 1>&2
 
 	output "Installing ora2pg."
-	wget --no-verbose https://github.com/darold/ora2pg/archive/refs/tags/v${ORA2PG_VERSION}.tar.gz 1>&2
-	tar -xf v${ORA2PG_VERSION}.tar.gz 1>&2
-	cd ora2pg-${ORA2PG_VERSION}/
+	# wget --no-verbose https://github.com/darold/ora2pg/archive/refs/tags/v${ORA2PG_VERSION}.tar.gz 1>&2
```

Comment:
We're already installing v23.1. Shouldn't that be enough?

Reply:
As i said in commit msg, the corresponding change has been introduced later than this release, around May. If we go with `wget` it will be using an old code?